docs/whitepaper.tex

```diff
@@ -504,72 +504,229 @@ \subsection*{Config UTxO}
+	\item The transaction should reference the valid config utxo
+	\item A seed utxo should be chosen
+	\item The seed utxo is spent
+	\item The seed utxo is passed to the pool id minting policy via the redeemer
+	\item the pool id is the hash of the seed utxo
+	\item The number of issued liquidity tokens set in the pool's UTxO datum matches the number of liquidity tokens minted from the liquidity token minting policy.
+	\item The pool id NFT is sent to the pool
+\end{itemize}
+
+Any Tx mentioned in these tests should be the same as the valid pool open
+except for any contradicting details stated in that test.
+
+\begin{itemize}
+	\item Non admin can initialize a pool.
+		\\ To test this we should validate a valid pool open
 	\item Can't under pay for liquidity on open
+		\\ To test this we should fail to validate a 2 txs
+			\begin{itemize}
+				\item One where:
+					We only provide 90 of each token.
+					but report minting 100
+					and mint 100
+				\item One where:
+					we only provide 90 of each token
+					and in the datum report minting 90
+					but actually mint 100
+			\end{itemize}
 	\item Can't open pool with 0 liquidity
+		\\ To test this we should fail to validate a tx where
+			both amounts and the liquidity are zero
 	\item Can't open pool with 0 of either exchange token
-		%% This is sorta redundant cause the liquidity needs to be
-		%% wrong or 0
+		\\ To test this we should fail to validate two Txs
+		one where each currency amount is zero
 	\item A utxo without the config NFT can't be used to open a pool
+		\\ To test this we should
+		produce an identical config utxo without the NFT
+		and then fail to validate a tx
+		where the invalid config utxo is referenced instead
 	\item Can't mint more than 1 token
+		\\ To test this we should fail to validate a tx where
+		two pool id tokens are minted
 	\item Can't send identifier token to non-pool address
+		\\ To test this we should fail to validate a tx where
+		the pool id NFT is kept by the user
+	\item Can't mint pool with wrong pool id
+		\\ To test this we should fail to validate a tx where
+		the token name of the minted id is ``aaaa"
 	\item Can't mint without spending seed Tx
-	\item Can't issue more liquidity than datum reflects
-	\item Can't mint with liquidity not equal to the initial invariant
-	\item Can't remove pool id from address during any type of transaction
-	\item Can't kill a pool with a swap
-	\item Can't kill a pool with a liquidity change
-	\item Can't swap on a dead pool
-	\item Can't add liquidity to a dead pool
-	\item Can't make unbalanced withdrawals from a dead pool
+		\\ To test this we should fail to validate a tx where
+		the seed utxo is not spent
 \end{itemize}
 
 \subsection*{Pool Address Validator tests}
 
+Terms
+
+A Valid pool refers to
+A pool as created by the ``valid pool open" described in the section above
+except with initial balances of $2,000,000$ and $1,000,000$.
+meaning the issued liquidity is $1,414,214$.
```

Comment:
Shouldn't this be `1732.05080757` = the square root of 2m of A * 3m of B?

Also do you mean the total issues liquidity tokens?